DuckDuckGo/UserText.swift

```diff
@@ -309,4 +309,15 @@ public struct UserText {
     public static let defaultBrowserHomeMessageSubheader = NSLocalizedString("home.message.subheader", value: "Open links with peace of mind, every time.", comment: "")
     public static let defaultBrowserHomeMessageTopText = NSLocalizedString("home.message.topText", value: "NEW IN IOS 14", comment: "")
     public static let defaultBrowserHomeMessageButtonText = NSLocalizedString("home.message.buttonText", value: "Set as Default Browser", comment: "")
+    
+    public static let doNotSellInfoText = NSLocalizedString("donotsell.info.headertext", value: """
```

Comment:
@bwaresiak - is it OK to use multiline text like this from the translation perspective, or should it use new line characters?

Reply:
That's a good question - I haven't tested that. 
@SlayterDev Can you / Did you regenerate Localizable.strings file? You can do that by running command `find DuckDuckGo/ -name "*.swift" -print0 | xargs -0 xcrun extractLocStrings -o DuckDuckGo/en.lproj` (run in the main project dir) - this is automated on my branch but still as work in progress. Sorry for the pain!